src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx

```diff
@@ -52,6 +52,75 @@ The Order and Usage Plan may contain defined terms that are denoted by capitaliz
     * In the event that a Customer indicates in an Order that it requires a purchase order (“PO”) for its subscription, Customer agrees to provide the required PO prior to the provisioning of the Products. If a Customer does not indicate a PO is required, Customer agrees that New Relic may issue invoice(s) and is entitled to such payment without a PO reference.
     * All Additional Usage fees will be invoiced regardless of a PO requirement.
     * The Product(s) are deemed accepted upon its provisioning.
+    * Any individual user authorized by Customer may accept terms on behalf of Customer.
+
+
+### Billable user terms [#billable-user-terms]
+
+This table explains how the billable user terms used in the Order relate to the terms used in the product UI and public docs: 
+
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        If your Order (as amended) has...
+      </th>
+
+      <th>
+        Then your [version of New Relic One pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#pricing-versions) is...
+      </th>
+      <th>
+        UI and docs refer to these users as
+      </th>      
+      <th>
+        Unit of Measure
+      </th>   
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        User
+      </td>
+
+      <td>
+      New Relic One **without** core users
+      </td>
+      <td>
+        full platform users
+      </td>
+      <td>
+        Monthly Provisioned User
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Full Platform User and Core User
+      </td>
+
+      <td>
+      Primary version of New Relic One: **with** core users
+      </td>
+      <td>
+      full platform user, and core user
+      </td>
+      <td>
+      Full Platform User and Core User
+      </td>
+    </tr>
+
+  </tbody>
+</table>
+
+Here are a couple examples of billable user term used in the Order and how they relate to other billing aspects:
+
+![Order form billable user terms](./images/new-relic-one-pricing-order-billable-users-terms.png "Order form billable user terms")
```

Comment:
Looks like the file name doesn't make `users` plural, this should get the image working!
```suggestion
![Order form billable user terms](./images/new-relic-one-pricing-order-billable-user-terms.png "Order form billable user terms")
```

Reply:
ah damn. sorry about that. thanks